opm/io/eclipse/EclFile.hpp

```diff
@@ -52,7 +52,7 @@ class EclFile
       char_array.clear();
     }
 
-    using EclEntry = std::tuple<std::string, eclArrType, int>;
+    using EclEntry = std::tuple<std::string, eclArrType, long int>;
```

Comment:
Instead of using `long int` here I would suggest using one of the fix size types like `int64_t` or `uint64_t`. The size of a `long int` is *not* standardized and e.g. on Windows `long int` is 32 bit even on a 64 bit computer.

Reply:
I would just use std::size_t which is big enough to represent the size of any objects in bytes.

Reply:
Is this still valid ? Notice that I'm no longer using the 20 byte header which included an 8 byte integer. Instead I've added the X231 format extensions which involved a pair of 16 byte headers (these includes a standard 4 byte integer).

long int and unsigned long int is used quite extensively in the existing version of the EclIO classes. 

If we should change from long int -> int64_t and from unsigned long int -> uint64_t, this should in my opinion be done in a separate PR in my opinion.

Reply:
I think it still makes sense to make this change - as long as we are only addressing memory - and not an on-disk format I agree that `std::size_t` is a good choice. I agree that the change can be in a a later PR.